### ISSUE TITLE
Pressing two keys at 'Set' before the start doesn't work offline

### ISSUE THREAD

**Issue by kimden** [supertuxkart/stk-code#5198]
## Steps to reproduce

1. Go to an **offline** game with Ready-Set-Go phase (that is, not artist debug 0 bots 0 ghosts)
2. Press two keys at the start (e.g. left and backwards)
3. Kart doesn't move after the start

## Description
The issue is not with my keyboard, it can press whatever number of keys at the same time. Also, repeating that online makes the kart move.

## Configuration

STK release version: ac376593 + custom unrelated commits (sorry if it happens to be my issue, in that case I'll make sure and close)

Keyboard: Bloody S98

**Comment by Alayan-stk-2** [comment]
This issue is specifically when pressing any another key alongside the backwards key, yes?

I can reproduce the issue pressing another key (even one not assigned to anything) + the backwards key ; but with the forward key I notice no issue (hence the 'minor' label).

**Comment by kimden** [comment]
Yes, I tested it like that, and I suppose it doesn't cover cases when the forward key + something else is pressed.

However, now I cannot reproduce it with all keys, e.g. left+backwards works now, but right+backwards doesn't.

**Comment by Alayan-stk-2** [comment]
Testing:
- In the master branch, left + backwards doesn't work properly (right + backwards does)
- In the Evolution branch, I cannot reproduce the issue at all

I'm not sure exactly which commit fixed it or how, but I'm assuming some of the changes around inputs and the start sequence did fix it inadvertently.
